### PR TITLE
[vcpkg] Use a crypto RNG to generate admin passwords

### PR DESCRIPTION
**Pull request by BillyONeal**
@strega-nil asked for this before; also fixes SDL policy.

### CODE REVIEW

File: scripts/azure-pipelines/linux/create-vmss.ps1 [hunk 1]
```diff
@@ -83,23 +83,40 @@ function Find-ResourceGroupName {
...
+  if ($chars.Length -ne 64) {
+    throw 'Bad chars length'
+  }
+
+  [Byte[]]$randomData = [Byte[]]::new($Length)
+  $rng = $null
+  try {
+    $rng = [System.Security.Cryptography.RandomNumberGenerator]::Create()
+    $rng.GetBytes($randomData)
+  }
```
Comment (strega-nil) [inline scripts/azure-pipelines/linux/create-vmss.ps1:110]: Does this rethrow if you don't catch? Could you add a comment to that effect?

**Review by strega-nil — APPROVED** [review]
Edit: accidentally hit approve and not request changes.

File: scripts/azure-pipelines/linux/create-vmss.ps1 (hunk 1 above)
Comment (strega-nil) [inline scripts/azure-pipelines/linux/create-vmss.ps1:110]: Does this rethrow if you don't catch? Could you add a comment to that effect?

**Review by strega-nil — CHANGES_REQUESTED** [review]
Could you please move the `New-Password` function to a `utilities` module? Having two of the exact same function copy-pastaed isn't great. Otherwise, thank you so much :D

File: scripts/azure-pipelines/linux/create-vmss.ps1 (hunk 1 above)
Comment (BillyONeal) [reply scripts/azure-pipelines/linux/create-vmss.ps1:110]: I don't really want to add 'this is how the language works' comments though...

Comment (strega-nil) [reply scripts/azure-pipelines/linux/create-vmss.ps1]: hmm, fair enough. I'm used to `try` meaning `catch`

**Review by strega-nil — APPROVED** [review]

**Pull request merged**